Gemfile.lock

```diff
@@ -330,3 +330,6 @@ DEPENDENCIES
   uk_postcode (~> 1.0.1)
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
+
+BUNDLED WITH
+   1.10.6
```

Comment:
We had a discussion about this in another PR and decided not to commit it as part of other, unrelated changes 
https://github.com/alphagov/smart-answers/pull/1825#discussion_r34886514